src/ensembl/src/shared/components/help-article/HistoryButtons.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { MutableRefObject } from 'react';
+import classNames from 'classnames';
+import HelpPopupHistory from '../help-popup/helpPopupHistory';
```

Comment:
The history, conceptually, belongs to the same folder as the history buttons. Especially if the history is going to be reused in the Help app.

Reply:
from code review: bring back HistoryButton component into help-popup directory. We will take care of reorganising the files in a different ticket as it needs more discussion/thinking.